### PR TITLE
Update bug template to ask for SF EXTENSION version

### PR DESCRIPTION
**Pull request by smaddox-sf**
we often have to ask a follow-up question to see which extension version they are on.  Changing the template to ask that upfront.  if we need to also know the VS Code version, we will ask in follow-ups and consider adding that to the template if we have to ask a lot

### CODE REVIEW

**Review by brpowell — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ---
 name: Bug report
 about: Create a report to help us improve
+title: ''
```
Comment (jag-sfdc) [inline .github/ISSUE_TEMPLATE/Bug_report.md:4]: How are these fields used?

Comment (smaddox-sf) [reply .github/ISSUE_TEMPLATE/Bug_report.md:4]: @jag-sfdc - This is the template users see when they create a new issue using the "bug" template - https://github.com/forcedotcom/salesforcedx-vscode/issues/new?assignees=&labels=&template=Bug_report.md

the change is to ask for the SF extension version vs. the VS Code version

Comment (jag-sfdc) [reply .github/ISSUE_TEMPLATE/Bug_report.md:4]: yes, I got that part. I didn't understand what these "title", "labels" and "assignees" fields are.

Comment (smaddox-sf) [reply .github/ISSUE_TEMPLATE/Bug_report.md:4]: not sure.  i think it's just a standard part of VS Code templates for issues?  i didn't change anything but the one label...

Comment (jag-sfdc) [reply .github/ISSUE_TEMPLATE/Bug_report.md:4]: @brpowell - Do you know what these fields are for? When the markdown is rendered, they appear as a table.
Screen Shot 2021-06-30 at 9 14 45 AM

Comment (brpowell) [reply .github/ISSUE_TEMPLATE/Bug_report.md:4]: I believe they are a mechanism to automatically apply labels/assignees based on the template. If they weren't on purpose Stephanie you can remove them.

Comment (xyc) [reply .github/ISSUE_TEMPLATE/Bug_report.md:4]: I think this is auto-added frontmatter https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

we can keep this.

**Review by jag-sfdc — APPROVED** [review]

**Pull request merged**